### PR TITLE
Router

### PR DESCRIPTION
**Pull request by willdyess**

### CODE REVIEW

File: src/app/demo-md/demo-md.component.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { Component, OnInit } from '@angular/core';
+import { Observable } from 'rxjs/Observable'
+import 'rxjs/add/observable/of'
+import { Subject } from 'rxjs/Subject'
+import {BehaviorSubject} from 'rxjs/BehaviorSubject'
```
Comment (sebastiankr) [inline src/app/demo-md/demo-md.component.ts]: same spacing

File: src/app/docs-layout/docs-layout.component.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
+
+
```
Comment (sebastiankr) [inline src/app/docs-layout/docs-layout.component.html]: clean this spacing

File: src/app/ea-website/ea-website.component.css [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+        position: absolute;
+      }
+      span.version {
+        font-size: 10px;
+        font-weight: 200;
+      }
+
+      .github-logo {
+        width: 20px;
+      }
```
Comment (sebastiankr) [inline src/app/ea-website/ea-website.component.css]: .github again?

File: src/app/fsm/fsm.component.html [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+        The model can transition <code>`from`</code> one state <span>`to`</span> another.
+
+          It is only do so if it can pass the <span>`gaurds`</span> in place.
+
+          <span>`onEntry`</span> and <span>`onExit`</span> events capture when the model is entering a state, or existing a state.
+
+          When the model enters one <span>`state`</span> there is an <span>`action`</span> that must be carried out prior to the exit event.
+
+          The <span>`gaurds`</span> are checked when the <span>`onExit`</span> event has occured prior to <span>`onEntry`</span>.
+
```
Comment (sebastiankr) [inline src/app/fsm/fsm.component.html]: markdown?

File: src/app/app.component.css [hunk 5]
```diff
@@ -0,0 +1,3 @@
+body {
+  margin:0;
+}
```
[the remark this answers is not in the document]
Comment (willdyess) [reply src/app/app.component.css]: Taken care of

**Pull request merged**